rinex/src/python.rs

```diff
@@ -0,0 +1,26 @@
+use pyo3::{exceptions::PyException, prelude::*};
+
+use crate::{
+    prelude::*,
+    observation::{
+        Crinex,
+        record::*,
+    },
+};
+
+#[pymodule]
+fn rinex(_py: Python, m: &PyModule) -> PyResult<()> {
+    /*
+     * prelude module
+     */
+    m.add_class::<Epoch>()?;
+    m.add_class::<EpochFlag>()?;
+    /*
```

Comment:
we should design python modules here, following the crate architecture. This would allow:

```python
from rinex import Epoch
from observation import ObservationData
```

Reply:
In the very near future, I will only focus on solving the wheel generator: ideally fix the build on MacOS and unlock the packaging in a release